src/main/kotlin/org/rust/lang/core/resolve2/FacadeResolve.kt

```diff
@@ -112,24 +112,19 @@ fun processMacros(
     }
     if (runBeforeResolve != null && runBeforeResolve()) return true
 
-    return modData.processMacros(scope, macroPath, processor, defMap, project)
+    return modData.processMacros(macroPath, processor, defMap, project)
 }
 
 private fun ModData.processMacros(
-    /** Must be not null during completion */
-    scope: RsMod?,
     /** null if path is qualified */
     macroPath: PsiElement?,
     processor: RsResolveProcessor,
     defMap: CrateDefMap,
     project: Project,
 ): Boolean {
-    val isCompletion = processor.name == null
-
     val isQualified = macroPath == null || macroPath is RsPath && macroPath.qualifier != null
     if (!isQualified) {
         check(macroPath != null)
-        val exactScopeVisibilityFilter = if (isCompletion) { mod: RsMod -> mod == scope } else { _ -> true }
```

Comment:
Error was here: `mod == scope`, instead we should use `mod == scope.originalElement`. But turns out that `mod` and `scope` are equal anyway when completing unqualified macro calls, so we can remove this visibility filter